lib/browser/remote/server.ts

```diff
@@ -259,6 +262,8 @@ const fakeConstructor = (constructor: Function, name: string) =>
 const unwrapArgs = function (sender: electron.WebContents, frameId: number, contextId: string, args: any[]) {
   const metaToValue = function (meta: MetaTypeFromRenderer): any {
     switch (meta.type) {
+      case 'nativeimage':
+        return electron.nativeImage.createFromBitmap(meta.value.buffer, meta.value.size as any);
```

Comment:
The `scaleFactor` option should also be passed, and we should add a way to pass the `isTemplateImage` information.

Reply:
If we're going to do a different serialization of native images here we should also update `type-utils`.

Reply:
@nornagon i was planning to handle that in a subsequent pr 🤔 i can do it here if preferable though.